src/hooks/useMintContractWithQuantity.tsx

```diff
@@ -57,7 +57,12 @@ export default function useMintContractWithQuantity({
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     async (contract: any) => {
       if (contract && address) {
-        return web3.utils.hexToNumber(await contract.balanceOfType(tokenId, address));
+        try {
+          const balance = await contract.balanceOfType(tokenId, address);
```

Comment:
putting the await inline caused this
![Screen Shot 2022-09-20 at 19 25 59](https://user-images.githubusercontent.com/80802871/191234466-788587b8-7cce-46e6-a2eb-3c7e5b06b970.png)

Reply:
wow, amazing catch!